lib/compilers/dotnet.ts

```diff
@@ -324,10 +324,7 @@ class DotNetCompiler extends BaseCompiler {
                 if (envVar) {
                     const [name] = envVar.split('=');
                     const normalizedName = name.trim().toUpperCase();
-                    if (
-                        normalizedName === 'DOTNET_JITDISASM' ||
-                        normalizedName === 'DOTNET_JITDISASMASSEMBILES'
-                    ) {
+                    if (normalizedName === 'DOTNET_JITDISASM' || normalizedName === 'DOTNET_JITDISASMASSEMBILES') {
```

Comment:
Why did this seemingly unrelated file change too? Can you undo this change unless it's strictly needed?

Reply:
I think this is our fault: some code got in that wasn't autoformatted... I'm going to fix this in main and then that will probably cause this to go away when you merge main! Sorry for the trouble.

Reply:
What do you think I should do then? Ignore it?